src/services/AbstractDFG.jl

```diff
@@ -444,13 +470,14 @@ Returns state of vertex data `.initialized` flag.
 
 Notes:
 - used by both factor graph variable and Bayes tree clique logic.
-TODO: Refactor
 """
 function isInitialized(var::DFGVariable; key::Symbol=:default)::Bool
-  return var.solverDataDict[key].initialized
+  solverData(var, key) && return solverData(var, key).initialized
```

Comment:
@GearsAD This causes an error

Reply:
data = solverData(var, key)
  data == nothing && return false
  return solverData(var, key).initialized

Reply:
Yeah I'm doing Python code accidentally again